server/infra/database/TransactionRepository.ts

```diff
@@ -0,0 +1,49 @@
+import Filter from 'interfaces/Filter';
```

Comment:
```suggestion
type Filter = Partial<{ token_id: string, wallet_id: string }>
```

Because #86 was merged, interfaces/Filter.ts no longer exists. You use this custom type to replace it.